### PR TITLE
add new external posts

### PR DESCRIPTION
**Pull request by hacktivist123**

### CODE REVIEW

File: src/posts/2023-05-23-cosmonic-os-netreap/index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+date: '2023-05-23T15:56:00.000Z'
+title: 'Cosmonic Builds the Ideal Container Toolset For Wasm at the Edge; Open Sources Netreap'
+ogImage: ogimage.png
+ogSummary: 'Cosmonic announce Netreap, a container toolset for Wasm at the Edge'
```
Comment (xmulligan) [inline src/posts/2023-05-23-cosmonic-os-netreap/index.md:5]: can you mention how this relates to Cilium?

File: src/posts/2023-05-23-deploying-isovalent-enterprise/index.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+date: '2023-05-23T15:56:00.000Z'
+title: 'Tutorial: Deploying Isovalent Enterprise for Cilium from Azure Marketplace'
+ogImage: ogimage.webp
+ogSummary: 'Learn about how to deploy Isovalent Enterprise for Cilium from Azure Marketplace'
```
Comment (xmulligan) [inline src/posts/2023-05-23-deploying-isovalent-enterprise/index.md:5]:
```suggestion
ogSummary: 'Learn how to deploy Isovalent Enterprise for Cilium from Azure Marketplace'
```

File: src/posts/2023-05-24-cilium-course/index.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+date: '2023-05-24T15:56:00.000Z'
+title: 'An inside look at the new Intro to Cilium course with course creator, Jef Spaleta!'
+ogImage: ogimage.png
+ogSummary: 'Hear about Cilium directly from Jef Spaleta, the creator of the new Intro to Cilium course'
```
Comment (xmulligan) [inline src/posts/2023-05-24-cilium-course/index.md:5]:
```suggestion
ogSummary: 'Hear why Jef Spaleta created the new Intro to Cilium course'
```

File: src/posts/2023-05-26-setup-cilium/index.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+---
+date: '2023-05-26T15:56:00.000Z'
+title: 'Setup Kubernetes + eBPF Cilium + Hubble with Performance Tuning'
+ogImage: ogimage.webp
+ogSummary: 'Deep dive into how to setup Kubernetes with eBPF cilium and Hubble'
```
Comment (xmulligan) [inline src/posts/2023-05-26-setup-cilium/index.md:5]:
```suggestion
ogSummary: 'Deep dive into how to setup Kubernetes with eBPF, Cilium, and Hubble'
```

File: src/posts/2023-05-27-race-conditions/index.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+---
+date: '2023-05-27T15:56:00.000Z'
+title: 'Race condition between kube-proxy and cilium'
+ogImage: ogimage.jpg
+ogSummary: 'Microsoft announce the general availability of Isivalent Cilium enterprise on the Azure marketplace'
```
Comment (xmulligan) [inline src/posts/2023-05-27-race-conditions/index.md:5]: This seems to be the wrong summary

File: src/posts/2023-05-30-isovent-cilium-azure/index.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+---
+date: '2023-05-30T15:56:00.000Z'
+title: 'General Availability: Isovalent Cilium Enterprise through Azure Marketplace'
+ogImage: ogimage.png
+ogSummary: 'Microsoft announce the general availability of Isivalent Cilium enterprise on the Azure marketplace'
```
Comment (xmulligan) [inline src/posts/2023-05-30-isovent-cilium-azure/index.md:5]:
```suggestion
ogSummary: 'Microsoft announce the general availability of Isovalent Cilium Enterprise on the Azure marketplace'
```

File: src/posts/2023-05-30-tietoevry/index.md [hunk 7]
```diff
@@ -0,0 +1,11 @@
+---
+date: '2023-05-30T15:56:00.000Z'
+title: 'Better policies, less tool sprawl: Tietoevry uses Cilium and Hubble'
+ogImage: ogimage.webp
+ogSummary: 'Learn about how Tietoevry uses Isovalent Enterprise for Cilium with Hubble to have advanced network policies'
```
Comment (xmulligan) [inline src/posts/2023-05-30-tietoevry/index.md:5]: This makes it sound like Hubble allows them to have network policy, can you rewrite to make it more clear?

File: src/posts/2023-05-27-race-conditions/index.md (hunk 5 above)
Comment (hacktivist123) [reply src/posts/2023-05-27-race-conditions/index.md:5]: yes, i mixed it up

File: src/posts/2023-05-26-cilium-mesh/index.md [hunk 8]
```diff
@@ -2,7 +2,7 @@
 date: '2023-05-26T15:56:00.000Z'
 title: 'Cilium Mesh: A new way to extend Kubernetes benefits across on-premises and cloud networking'
 ogImage: ogimage.jpeg
-ogSummary: 'Learn about how Cilium mesh has helped extend the benefits of Kubernetes'
+ogSummary: 'Learn about how Cilium Mesh provides a fresh way to maximize the advantages of Kubernetes, offering new possibilities for extending its benefits'
```
Comment (xmulligan) [inline src/posts/2023-05-26-cilium-mesh/index.md:5]: That's not exactly what Cilium Mesh is, can you reread the the launch blog post and this one to make sure you understand what it is doing?

File: src/posts/2023-05-30-tietoevry/index.md [hunk 9]
```diff
@@ -2,7 +2,7 @@
 date: '2023-05-30T15:56:00.000Z'
 title: 'Better policies, less tool sprawl: Tietoevry uses Cilium and Hubble'
 ogImage: ogimage.webp
-ogSummary: 'Learn about how Tietoevry uses Isovalent Enterprise for Cilium with Hubble to have advanced network policies'
+ogSummary: 'Learn how Tietoevry uses Hubble for observability and Isovalent Enterprise for Cilium to implement advanced network policies'
```
Comment (xmulligan) [inline src/posts/2023-05-30-tietoevry/index.md:5]: Hubble is a part of Isovalent Enterprise for Cilium, this makes it seem like they are separate things

**Comment by hacktivist123** [comment]
Okay but the summary is based on how the article is structured. Did you read that particular article?

**Comment by xmulligan** [comment]
Yes, I've read the article. It talks about "Kubernetes-style principles powering the network layer has a range of benefits". The benefits aren't from Kubernetes itself, but rather from Cilium which is built in the Kubernetes/cloud native style. It's a subtle difference, but an important one. Cilium providing the networking in a Kubernetes-style way, extends the benefits of cloud native networking outside the Kubernetes world.

The way you currently have it written makes it seem like Cilium Mesh is just extending Kubernetes beyond Kubernetes.

File: src/posts/2023-05-26-cilium-mesh/index.md [hunk 10]
```diff
@@ -2,7 +2,7 @@
 date: '2023-05-26T15:56:00.000Z'
 title: 'Cilium Mesh: A new way to extend Kubernetes benefits across on-premises and cloud networking'
 ogImage: ogimage.jpeg
-ogSummary: 'Learn about how Cilium Mesh provides a fresh way to maximize the advantages of Kubernetes, offering new possibilities for extending its benefits'
+ogSummary: 'Learn about how Cilium Mesh helps enterprises bridge their on-premises, cloud or edge networking assets into a cloud-native world'
```
Comment (xmulligan) [inline src/posts/2023-05-26-cilium-mesh/index.md:5]:
```suggestion
ogSummary: 'Learn about how Cilium Mesh helps enterprises bridge their on-premises, cloud or edge networking assets into a cloud native world'
```

**Pull request merged**